styles/hydrogen.less

```diff
@@ -373,3 +366,17 @@ svg:first-child {
     color: @text-color-info !important;
   }
 }
+
+}
+
+// -------------- Overwrite Autocomplete-Plus style -----------------
+
+.autocomplete-plus {
```

Comment:
Is this still inside of .hydrogen? If not because it applies to the editor, can you explain why we are setting the max width is it off the page because if it is just really long but not off the page, maybe we should have it?

Reply:
This is not inside `.hydrogen` scope, since we can't tweak autocompletion styles otherwise.

I put `max-width` style here since completions from Jupyter kernels are kinda non-formatted (we can use whatever kernel as far as it follows jupyter messaging protocol, and there is no general docstring style for them).
It means, if we don't have this style, it looks really weird, like autocompletion expanding in a silly way.

Reply:
Well I am not sure how my styles to the watches autocomplete will affect the global scope. Be aware this could cause styling issues. For now I am ok with this change however.